### PR TITLE
Update appearance of 'Jenkins is starting' pages

### PR DESCRIPTION
**Pull request by janfaracik**
This PR updates the appearance of the 'Jenkins is starting' pages. The pages are now slightly more padded, content has been refined, use the [<l:spinner/>](https://weekly.ci.jenkins.io/design-library/Progress/) component (rather than a custom one) and have a soft animation on load.

### Booting

**Before**
image

**After**
image

### Safe restart

**Before**
image

**After**
image

### Restart

**Before**
image

**After**
image

⚠️ Unsure of how to access `core/src/main/resources/hudson/lifecycle/WindowsInstallerLink/_restart.jelly`

### Testing done

* Pages appear as expected

### Proposed changelog entries

- Update appearance of 'Jenkins is starting' pages.

### Proposed upgrade guidelines

N/A

```[tasklist]
### Submitter checklist
- [x] The Jira issue, if it exists, is well-described.
- [x] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)). Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [x] There is automated testing or an explanation as to why this change has no tests.
- [x] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [x] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [x] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [x] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [x] For new APIs and extension points, there is a link to at least one consumer.
```

### Desired reviewers

@jenkinsci/sig-ux 

Before the changes are marked as `ready-for-merge`:

```[tasklist]
### Maintainer checklist
- [x] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [x] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [x] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [x] Proper changelog labels are set so that the changelog can be generated automatically.
- [x] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [x] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).
```

### CODE REVIEW

**Comment by timja** [comment]
You can do:

/label web-ui,rfe
/reviewer jenkinsci/sig-ux

File: core/src/main/resources/hudson/util/HudsonIsRestarting/index_de.properties
Comment (timja) [inline core/src/main/resources/hudson/util/HudsonIsRestarting/index_de.properties:1]: When changing the English text we should update / remove translations. 

e.g. here I think you would drop the `Bitte warten`

File: core/src/main/resources/hudson/util/HudsonIsLoading/index_pt_BR.properties
Comment (timja) [inline core/src/main/resources/hudson/util/HudsonIsLoading/index_pt_BR.properties:23]: This needs updating still (capturing as outstanding)

Comment (janfaracik) [reply core/src/main/resources/hudson/util/HudsonIsLoading/index_pt_BR.properties]: Done, updated the translations that I could, removed those that I couldn't

**Review by timja — DISMISSED** [review]
LGTM

File: core/src/main/resources/hudson/util/HudsonIsRestarting/index_de.properties
Comment (uhafner) [reply core/src/main/resources/hudson/util/HudsonIsRestarting/index_de.properties:1]: I can confirm, it should be "Jenkins wird neu gestartet"

Comment (timja) [reply core/src/main/resources/hudson/util/HudsonIsRestarting/index_de.properties:1]:
```suggestion
Please\ wait\ while\ Jenkins\ is\ restarting=Jenkins wird neu gestartet.
```

File: core/src/main/resources/hudson/util/HudsonIsRestarting/index_da.properties
Comment (timja) [inline core/src/main/resources/hudson/util/HudsonIsRestarting/index_da.properties:24]: there's still a number of translations outstanding for this one `Please\ wait\ while\ Jenkins\ is\ restarting`

Either update or remove translation please

File: core/src/main/resources/hudson/util/HudsonIsRestarting/index_de.properties
Comment (uhafner) [reply core/src/main/resources/hudson/util/HudsonIsRestarting/index_de.properties:1]: Should the "." be part of the text or not?

**Review by timja — APPROVED** [review]
LGTM thanks!

**Review by StefanSpieker — APPROVED** [review]

**Comment by timja** [comment]
/label ready-for-merge

----

This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

File: core/src/main/resources/hudson/util/HudsonIsLoading/index.jelly
Comment (daniel-beck) [inline core/src/main/resources/hudson/util/HudsonIsLoading/index.jelly:54]: Why remove the trailing period? The other line is (or was) a title so didn't need it, but this one isn't.

Comment (janfaracik) [reply core/src/main/resources/hudson/util/HudsonIsLoading/index.jelly:54]: Personal preference, I don't personally see the value of them for things like this. Would be a good thing to standardise across Jenkins, as right now many fields include periods, many don't.

**Review by MarkEWaite — APPROVED** [review]
Worked well in my interactive testing

File: core/src/main/resources/hudson/util/HudsonIsLoading/index.jelly
Comment (daniel-beck) [reply core/src/main/resources/hudson/util/HudsonIsLoading/index.jelly:54]: > many fields include periods, many don't

Are there many _sentences_ that don't have periods?

File: core/src/main/resources/hudson/util/HudsonIsRestarting/index_da.properties
Comment (daniel-beck) [inline core/src/main/resources/hudson/util/HudsonIsRestarting/index_da.properties:24]: Was this intentional? I don't speak the language but Google Translate says it's now "Please wait Jenkins restarts" which seems not great translated back.

**Pull request merged**
